File: app/src/main/java/com/pepcox/richtar/richtarjakub/data/BeerDatabase.java
package com.pepcox.richtar.richtarjakub.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {Beer.class}, version = 1)
public abstract class BeerDatabase extends RoomDatabase {
    public abstract BeerDao beerDao();
}

File: app/src/main/java/com/pepcox/richtar/richtarjakub/data/BeerDatabase.kt
package com.pepcox.richtar.richtarjakub.data

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

@Database(entities = arrayOf(Beer::class, BeerHistory::class), version = 2)
abstract class BeerDatabase : RoomDatabase() {
    abstract fun beerDao(): BeerDao
    abstract fun beerHistoryDao(): BeerHistoryDao
}
